### PR TITLE
DOC-1559: Fixes for core changes api section

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: DOC-1559

Description of Changes:
* Placeholder text

Pre-checks:
- [X] Branch prefixed with `feature/` or `hotfix/`
- [X] `_data/nav.yml` has been updated (if applicable)
- [X] Files has been included where required (if applicable)
- [X] Files removed have been deleted, not just excluded from the build (if applicable)
- [X] ~~(New product features only) Release Note added~~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc [hunk 1]
```diff
@@ -17,21 +17,21 @@ It also documents the {productname} 6.0 APIs that have been removed.
...
 The new `editor.options` API replaces two old APIs: `editor.settings` and `editor.getParam`.
 
-Along with this new API, the new `tinymce.activeEditor.options` registry replaces `tinymce.activeeditor.settings`.
+Along with this new API, the new `tinymce.activeEditor.options` API replaces `tinymce.activeEditor.settings`.
 
 [[editor-uploadImages]]
 === `editor.uploadImages()`
 
-The `editor.uploadImages()` API now includes a `remove` flag.
+The results returned from `editor.uploadImages()` API now includes a `removed` property.
```
Comment (metricjs) [inline modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc:32]:
```suggestion
The results returned from `editor.uploadImages()` API now include a `removed` property.
```

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc [hunk 2]
```diff
@@ -55,9 +53,9 @@ The dialog API’s `setData` method now uses a deep merge algorithm.
 Consequently, it now supports partial nested objects.
 
 [[promise]]
-==== `success` and `failure` callbacks replaced by `Promise` in multiple APIs
+=== `success` and `failure` callbacks replaced by `Promise` in multiple APIs
 
- Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs report resource loading status with `success` or `failure` callbacks.
+Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs reported resource loading status with `success` or `failure` callbacks.
 
 These APIs now, instead, return a `Promise`, with a value of either `resolve` or `reject`. Also, `EditorUpload`, which already returns a `Promise`, had callbacks removed.
```
Comment (metricjs) [inline modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc]:
```suggestion
These APIs now, instead, return a `Promise`. Also, `EditorUpload`, which already returns a `Promise`, had callbacks removed.
```

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc [hunk 3]
```diff
@@ -55,41 +53,41 @@ The dialog API’s `setData` method now uses a deep merge algorithm.
...
 
 [[disabled]]
-=== `disabled` functions renamed in all {productname} APIs
+=== `disabled` methods renamed in all {productname} APIs
 
-Previously, multiple {productname} APIs and UI classes used `disable` or `disabled` in their function or property names.
+Previously, multiple {productname} APIs and UI components used `disable` or `disabled` in their method or property names.
 
-In {productname} 6, these function names have been changed and configuration changes are required.
+In {productname} 6, these method names have been changed and configuration changes are required.
```
Comment (metricjs) [inline modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc:79]:
```suggestion
In {productname} 6, these method and property names have been changed and configuration changes are required.
```

**Review by metricjs — APPROVED** [review]

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc [hunk 4]
```diff
@@ -101,7 +99,7 @@ As well, the `DomParser` API no longer uses a custom parser internally and inste
 [[editor-setcontent]]
 === `editor.SetContent`
```
Comment (lnewson) [inline modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc:100]: This needs fixing.
```suggestion
=== `SetContent` editor event
```

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc (hunk 2 above)
Comment (ltrouton) [reply modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc]: Fix in a separate commit

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc [hunk 5]
```diff
@@ -55,41 +53,41 @@ The dialog API’s `setData` method now uses a deep merge algorithm.
 Consequently, it now supports partial nested objects.
 
 [[promise]]
-==== `success` and `failure` callbacks replaced by `Promise` in multiple APIs
+=== `success` and `failure` callbacks replaced by `Promise` in multiple APIs
 
- Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs report resource loading status with `success` or `failure` callbacks.
+Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs reported resource loading status with `success` or `failure` callbacks.
```
Comment (MitchCTiny) [inline modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc:58]: Should these API names be surrounded by ` tags?

Comment (ltrouton) [reply modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc]: Done

File: modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc (hunk 3 above)
Comment (ltrouton) [reply modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc]: I changed this section a little bit

**Pull request merged**